staff/carlos-diestro/cv-styled/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>John Doe Resume</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <main>
+    <header>
+      <div class="item-1">
+        <img src="https://via.placeholder.com/90x90" alt="John">
+      </div>
+      <div class="name item-2">
+        <h1>John Doe</h1>
+        <h3>Web Developer</h3>
+      </div>
+      <address class="contact item-3">
+        Roc Boronat, 35, 08005 Barcelona
+        <br>
+        <a href="https://github.com" target="_blank">github.com/jdoe</a>
```

Comment:
use ```<p>```s for newlines instead of br

Reply:
In this case it's fine. More info here https://developer.mozilla.org/es/docs/Web/HTML/Elemento/br